content/en/guides/os-commercial-ecosystem.md

```diff
@@ -0,0 +1,175 @@
+
+* **Foundations.** Foundations have had an enormous impact on the world of open source in recent years. Your organization can benefit from working with foundations ranging from The Linux Foundation to the Apache Foundation, and you may benefit from launching an open source-focused foundation. Understanding the role of open source foundations in fostering sustainable open source projects, and considering when and how to involve them, will help set you up for success.
+
+*“Foundations provide a lot of value. Without them it has historically been hard for a lot of very critical projects to get the funding they need to be well maintained. They help ensure a level playing field and they provide mechanisms for organizations to give back to open source projects without contributing developers directly.”* — Luke Faraone, Software Engineer, Dropbox
+
+&nbsp;
+
+# How to build an open source ecosystem
+
+To build out open source ecosystems around your projects, you must focus on the following: governance; Resources needed to use an OS project as a commercial dependency; Tracking progress, and making adjustments. Let’s look into each of these and the value they can bring to your commercial ecosystem strategy:
+
+## 1. Establish a healthy governance system
+
+It is essential to create a neutral structure that makes it easy for competitors to participate, and you should also consider whether external stakeholders should participate in the governance process. Your ecosystem is at its healthiest when governance is independent and receiving diverse contributions. 
+
+If you oversee the technical governance of your project, you may be far less qualified to make decisions regarding support, trademarking, or licensing. Likewise, you must separate technical and financial decision-making. Some of the best technical advancements take place in environments where moonshots are encouraged. Encourage technical contributors to aim for moonshots, and separate decision-makers can evaluate whether they are financially feasible.
+
+*“One of the things that we've tried to engender in the Kubernetes community is the idea of a project over people or company. What's good for the project is a separate question from what's good for the companies that are involved with the project. When you end up with open source projects that are too tightly tied to a single company, really sticky issues can arise.”* – Joe Beda, Co-Founder of Kubernetes, and Co-Founder and CTO of Heptio
+
+## 2. Evaluate resources
+
+When partners or customers are deciding whether to utilize open source projects in their businesses, there are certain different resources they might evaluate or consider:
```

Comment:
Should we have a bullet on documentation in this section? Or maybe we can move some information from the next section.